### PR TITLE
Create archivarius.yml

### PR DESCRIPTION
**Pull request by RussKie**
## Proposed changes

- Configure the bot's message

### Before

image

### After

image

## Test methodology 

- manual

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by sharwell** [comment]
I am passionately opposed to a bot validating the template. I frequently see a long template, tl;dr, delete the whole thing, and fill out something else I had in mind instead. I expect other users to do the same, and generally find that it does not inherently negatively impact the quality of an issue.

**Comment by gerhardol** [comment]
@sharwell 
See https://github.com/gitextensions/gitextensions/issues/6737 before edit, it only closes "empty" issues

File: .github/archivarius.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Comment to post when potential duplicates are detected
+referenceComment: |
+  Thank you for the bug report :+1:
+  
+  Sadly, it appears some important information is still missing, which you may have overlooked while submitting the bug.
+  Without this information we may not be able to assistance you in resolving the issue.
```
Comment (mstv) [inline .github/archivarius.yml]: Me again: My favourite dictionaries know "assistance" only as noun. They suggest either "to assist" or "to give / provide / render assistance".

Comment (RussKie) [reply .github/archivarius.yml]: > Me again: My favourite dictionaries know "assistance" only as noun. They suggest either "to assist" or "to give / provide / render assistance".

You're a great help, mate!
I've been agonising over this message and iterate a number of different variants. I completely missed this 🤦‍♂

**Comment by RussKie** [comment]
> I am passionately opposed to a bot validating the template. I frequently see a long template, tl;dr, delete the whole thing, and fill out something else I had in mind instead. I expect other users to do the same, and generally find that it does not inherently negatively impact the quality of an issue.

This is an intermediate state to reduce a number of incomplete NBug reports. I'm sure you have noticed we are NBug reports now (which is great), however users appear to ignore (purposely or not) requests for repro steps. Pretty much all follow up questions are ignored too.
Without the repro steps we can't meaningfully help/fix issues, so we close them.
The bot automates this manual work.

The bot only looks at issues that either have the following string: 
* `The sections below must be filled in and this text must be removed or the issue will be closed.` or 
* an empty "Steps to reproduce" section.

There are plans for v3.2 to enhance the NBug UI to request a user to supply steps to reproduce before an issue can be submitted.

File: .github/archivarius.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  
+  Please review your submission, fill the missing information in and ping us back to re-open the issue.
+  Thanks again
```
Comment (pmiossec) [inline .github/archivarius.yml]: Is "ping us back" enough clear for users (not necessarily used to github) to understand that they have to add a comment to do it?

Something like: 
`Please review your submission, fill the missing information in and post a new comment to ping us back to re-open the issue.`

**Pull request merged**